TestProject/API/EndPointItem.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  EndPointItem.swift
+//  TestProject
+//
+//  Created by propelrr-mac-pro on 6/15/22.
+//
+
+import Foundation
+import Alamofire
+
+enum EndPointItem {
+    
+    case register
+    case users
+    
+}
+
+extension EndPointItem: EndPointType {
+    
+    var baseURL: String {
+        switch self {
+        default: return "https://reqres.in/"
+        }
+    }
+    
+    var path: String {
+        switch self {
+        case .register:
+            return "api/register"
+        case .users:
+            return "api/users"
+        }
+    }
+    
+    var httpMethod: HTTPMethod {
+        switch self {
+        case .register:
+            return .post
+        case .users:
+            return .get
+        default:
+            return .get
+        }
+    }
+    
+    var headers: HTTPHeaders? {
+        switch self {
+        case .register:
```

Comment:
```suggestion
        case .register, .users:
```

You can compile multiple cases into a single block. :)